pom.xml

```diff
@@ -118,20 +115,24 @@
         <artifactId>Saxon-HE</artifactId>
         <version>10.7</version>
       </dependency>
+      <dependency>
+        <groupId>org.graalvm.sdk</groupId>
+        <artifactId>graal-sdk</artifactId>
+        <version>${graalvm.version}</version>
+      </dependency>
       <dependency>
         <groupId>org.graalvm.js</groupId>
         <artifactId>js</artifactId>
-        <version>22.1.0</version>
+	<version>${graalvm.version}</version>
+	<scope>runtime</scope>
       </dependency>
       <dependency>
         <groupId>org.graalvm.js</groupId>
         <artifactId>js-scriptengine</artifactId>
```

Comment:
Drop this dependency -- needed only if using the ScriptEngine API

Reply:
Tests fail without it.

Reply:
Weird, I dropped it on. my branch and they are passing fine.